webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -1346,6 +1366,7 @@ public static ByteArrayOutputStream requestsToZip(ServletContext context, String
         String safeGlobalTitle = toFileSafeString(globalTitle);
         String computerDisplayFileName = safeGlobalTitle + " - Computer Display PDFs";
 
+        ScrambleRequest[] scrambleRequests = GSON.fromJson(query.get("sheets"), ScrambleRequest[].class);
```

Comment:
As I commented over in `ScrambleHandler`, I think this sort of string parsing belongs up in the handler class, and not down in this method.

Reply:
Fixed